custom_components/tuya_local/devices/petsnowy_snowplus_catlitter.yaml

```diff
@@ -0,0 +1,214 @@
+name: Cat litter box
+products:
+  - id: bdfimkssp9ews36b
+    manufacturer: PetSnowy
+    model: Snow+
+entities:
+  - entity: switch
+    name: Auto clean
+    category: config
+    icon: "mdi:flower"
+    dps:
+      - id: 4
+        type: boolean
+        name: switch
+  - entity: number
+    name: Clean wait time
+    category: config
+    class: duration
+    icon: "mdi:timer"
+    dps:
+      - id: 5
+        type: integer
+        name: value
+        unit: min
+        range:
+          min: 2
+          max: 60
+        mapping:
+          - step: 2
+  - entity: sensor
+    name: Cat weight
+    category: diagnostic
+    class: weight
+    dps:
+      - id: 6
+        name: sensor
+        type: integer
+        unit: g
+        class: measurement
+        range:

```

Comment:
range is not needed for sensors in general, unless there is a transformation that requires it (like invert, or target__range)